### PR TITLE
Removed font size multiplier

### PR DESCRIPTION
**Pull request by dee**
On Windows, IDE fonts in a project tree, editor tabs etc are noticeably larger than a system font. That was because an IDE font was 20% bigger than a system one. Tested it on 4K and 1080p.

### CODE REVIEW

File: Plugin/clScrolledPanel.cpp
Comment (Jarod42) [inline Plugin/clScrolledPanel.cpp:388]: So whole `__WXMSW__` block can be removed

Comment (dee) [reply Plugin/clScrolledPanel.cpp]: Something like this, i guess.

Comment (Jarod42) [reply Plugin/clScrolledPanel.cpp]: No need to set same value for other, simply
```
wxFont clScrolledPanel::GetDefaultFont()
{
    wxFont f = wxSystemSettings::GetFont(wxSYS_DEFAULT_GUI_FONT);
#if defined(__WXMAC__)
    float pointSize = f.GetFractionalPointSize() * 1.2;
    f.SetFractionalPointSize(pointSize);
#endif
    return f;
}
```
or
```
wxFont clScrolledPanel::GetDefaultFont()
{
    wxFont f = wxSystemSettings::GetFont(wxSYS_DEFAULT_GUI_FONT);
#if defined(__WXMAC__)
    f.SetFractionalPointSize(1.2 * f.GetFractionalPointSize());
#endif
    return f;
}
```

Comment (dee) [reply Plugin/clScrolledPanel.cpp]: Yes, really, I am setting the same value back.

**Review by Jarod42 — APPROVED** [review]

**Comment by dee** [comment]
Thank you!

**Comment by Jarod42** [comment]
Unsure why you close your PR.

Note that:
- approving != merging.
- I don't have write access, I'm just a contributor like you.

**Comment by dee** [comment]
My bad, I thought it was finished.

**Comment by eranif** [comment]
Even though some CI are failing - these were fixed in master branch. Once the Ubuntu build is OK, I will merge it

**Pull request merged**
